src/components/addressAutocomplete/components/EmptySuggestion.tsx

```diff
@@ -1,18 +1,16 @@
-import React from 'react';
+import styled from 'styled-components';
+
+const defaultEmptySuggestionText = 'Aucune adresse trouvée :(';
+
+const EmptySuggestionWrapper = styled.p`
+  margin: 0;
+  color: '#454545';
+  padding: '0.25rem 1rem 0.75rem 1rem';
+  font-style: 'italic';
+`;
 
 const EmptySuggestion: React.FC<{ text?: string }> = ({
```

Comment:
This component is not very usefull, can we put it in the `AddressAutocomplete` style directly ?

Reply:
Sure !